### DISCUSSION TITLE
Verilator support for SVUnit

### DISCUSSION THREAD

**Discussion by svenka3** [svunit/svunit#222] [Show and tell]
Motivated by recent updates to Verilator (devel), I attempted a port of SVUnit to Verilator and I have one basic example running. 

https://github.com/svenka3/svunit/tree/srini_verilator

There are few key issues including:

1. Issue with "filter" - need more digging
2. find with item.. not working fine
3. Few unsupported features
4. One (or 2 maybe) code enhancements as suggested by Veriltor Lint - can be looked at

Open to all comments and feedback.

Regards
Srini

**Comment by rpjayaraman** [comment]
Hi Srini, raised a pull request https://github.com/svunit/svunit/pull/223
